### ISSUE TITLE
Accuracy of EMA is even lower than the trained model

### ISSUE THREAD

**Issue by Shiweiliuiiiiiii** [DingXiaoH/RepLKNet-pytorch#24]
Hi Authors,

Thanks again for your excellent work. I am using your structure and running some experiments, but found that the accuracy of the EMA model is 1% lower than the original trained model. Have you noticed this problem? Or I have missed some important implementation about this. 

Thanks,
Shiwei

**Comment by TheGreatTreatsby** [comment]
> Hi Authors,
> 
> Thanks again for your excellent work. I am using your structure and running some experiments, but found that the accuracy of the EMA model is 1% lower than the original trained model. Have you noticed this problem? Or I have missed some important implementation about this.
> 
> Thanks, Shiwei
hi，I met the same problem.I wonder what's your training input size?and the number of the picture in your dataset?

**Comment by TheGreatTreatsby** [comment]
I wonder the input size when you train,and how much pictures in your dataset?

**Comment by TheGreatTreatsby** [comment]
> Hi Authors,
> 
> Thanks again for your excellent work. I am using your structure and running some experiments, but found that the accuracy of the EMA model is 1% lower than the original trained model. Have you noticed this problem? Or I have missed some important implementation about this.
> 
> Thanks, Shiwei

**Comment by DingXiaoH** [comment]
When you train a model from scratch , the accuracy of EMA may be even lower than the trained model, which is expected. EMA usually works while finetuning a model.
